content/en/bits_ai/bits_ai_dev_agent/_index.md

```diff
@@ -27,14 +27,41 @@ Bits AI Dev Agent is available for the following Datadog products:
 | [Code Security][2]        | Preview              | Remediates code vulnerabilities individually or in bulk    |
 | [Test Optimization][4]    | Preview              | Provides code fixes for flaky tests and verifies that tests remain stable                                                         |
 | [Continuous Profiler][3]  | Preview              | Provides code changes for [Automated Analysis][10] insights                  |
-| [Containers][12  ]        | Preview              | Provides code changes for Container Recommendations                  |
+| [Containers][12]          | Preview              | Provides code changes for Container Recommendations                  |
 
 **Note**: Enabling Bits AI Dev Agent is product-specific. Even if it's active for one Datadog product, it must be separately enabled for each additional product you use.
 
 ## Key capabilities 
 
 The following sections detail how Bits AI Dev Agent integrates with Datadog products to generate contextual code fixes.
 
+### Pull request assistance
+
+Bits AI Dev Agent integrates with GitHub to create pull requests, respond to comments, update commits, and fix CI failures. 
+
+- Generates PR titles and descriptions based on your PR template.
+- Opens PRs as drafts, iterates using CI logs, and marks them ready for review when checks pass.
+- Continues iterating in response to chat messages and review feedback.
+  
+  **Note**: Comment `@Datadog` to prompt Bits for updates to the PR. Bits Dev never auto-merges PRs.
+
+Go to **Bits AI** > **Dev Agent** > **[Code sessions][7]** to see all Dev Agent code sessions and generated PRs. You can search sessions and filter by service, product source, and status.
+
+### Auto-push
+
+Auto-push allows the Agent to create branches, push code, and open PRs when it detects something it can help you with. For example, the Agent can:
```

Comment:
Just wanted to check - sometimes we refer to this as "Bits" or "the Agent". Do we use them interchangeably?